go.mod

```diff
@@ -29,7 +29,11 @@ require (
 	google.golang.org/grpc v1.58.2
 )
 
-require github.com/Microsoft/hcsshim v0.11.0 // indirect
+require (
```

Comment:
i am not sure why go adds a new section sometimes, can we combine with the rest of the indirect deps?